community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -214,16 +275,16 @@ class QueryPlanTest extends DocumentingTestBase {
     )
   }
 
-  @Test def expand() {
+  @Test def expandAll() {
```

Comment:
I think expandInto makes a lot of sense having here. It's a cute and weird little animal, and would benefit from some docs love